test/config-next/sa.json

```diff
@@ -24,7 +24,8 @@
       ]
     },
     "features": {
-      "AllowAccountDeactivation": true
+      "AllowAccountDeactivation": true,
+      "UsePrefixDB": true
```

Comment:
This should also be set in anything that calls `sa.NewDBMap`, so `ocsp-updater`, `ocsp-responder`, `notify-mailer`, `cert-checker`, `expired-authz-purger`, `admin-revoker`, `contact-exporter`, and probably something else I'm forgetting.